doc-files/release-notes-13.md

```diff
@@ -0,0 +1,80 @@
+[JDK-8219362](https://bugs.openjdk.java.net/browse/JDK-8219362)|Update to 608.1 version of WebKit|web
+[JDK-8217942](https://bugs.openjdk.java.net/browse/JDK-8217942)|Upgrade to libxslt 1.1.33|web
+[JDK-8222912](https://bugs.openjdk.java.net/browse/JDK-8222912)|Websocket client doesn't work in WebView|web
+[JDK-8221941](https://bugs.openjdk.java.net/browse/JDK-8221941)|Wrong package declaration for WCTextRunImpl.java in web module|web
+[JDK-8219734](https://bugs.openjdk.java.net/browse/JDK-8219734)|[WebView] Get rid of macOS SDK private API usage|web
+[JDK-8219917](https://bugs.openjdk.java.net/browse/JDK-8219917)|[WebView] Sub-resource integrity check fails on Windows and Linux|web
+[JDK-8227431](https://bugs.openjdk.java.net/browse/JDK-8227431)|[Windows] Fix assertion failure on X86 32-bit when enabling CLOOP based JavaScript interpreter|web
+[JDK-8230361](https://bugs.openjdk.java.net/browse/JDK-8230361)|[web] Cookies are not enabled in WebKit v608.1 |web
+[JDK-8229328](https://bugs.openjdk.java.net/browse/JDK-8229328)|[windows] PlatformFileHandle type should be JGObject rather than void 
+[JDK-8222788](https://bugs.openjdk.java.net/browse/JDK-8222788)|javafx.web build fails on XCode 10.2|web
+[JDK-8226951](https://bugs.openjdk.java.net/browse/JDK-8226951)|Backout commit for JDK-8226537 to fix the attribution|window-toolkit
+[JDK-8211302](https://bugs.openjdk.java.net/browse/JDK-8211302)|DragAndDrop no longer works with GTK3|window-toolkit
+[JDK-8226537](https://bugs.openjdk.java.net/browse/JDK-8226537)|Multi-level Stage::initOwner can crash gnome-shell or X.org server|window-toolkit
+[JDK-8226274](https://bugs.openjdk.java.net/browse/JDK-8226274)|NPE in WinWindow.notifyMoving when Stage with no Scene is shown on 2nd monitor|window-toolkit
+[JDK-8088717](https://bugs.openjdk.java.net/browse/JDK-8088717)|Win: UNDECORATED windows are not minimized with the taskbar button|window-toolkit
+[JDK-8220272](https://bugs.openjdk.java.net/browse/JDK-8220272)|Window order is not correct when Modality.WINDOW_MODAL|window-toolkit
+[JDK-8212060](https://bugs.openjdk.java.net/browse/JDK-8212060)|[GTK3] Stage sometimes shown at top-left before moving to correct position|window-toolkit
+
+
+## List of Enhancements
+
+Issue key|Summary|Subcomponent
+---------|-------|------------
+[JDK-8226454](https://bugs.openjdk.java.net/browse/JDK-8226454)|Point2D and Point3D should implement Interpolatable|animation
+[JDK-8221269](https://bugs.openjdk.java.net/browse/JDK-8221269)|Extract embedded actions from JSL grammar file to Visitor class|build
+[JDK-8223760](https://bugs.openjdk.java.net/browse/JDK-8223760)|support static build for macosx|build
+[JDK-8222258](https://bugs.openjdk.java.net/browse/JDK-8222258)|Add exclusion scope for LightBase|graphics
+[JDK-8167148](https://bugs.openjdk.java.net/browse/JDK-8167148)|Add native rendering support  by supporting WritableImages backed by NIO ByteBuffers|graphics
```

Comment:
Somehow there's an extra space in "support  by".

Reply:
Oops. This one is my fault...I see that it was in the data I pulled from JBS and sent to Joeri and Johan.